scripts/roll_release.sh

```diff
@@ -52,6 +52,14 @@ echo "Sanity checking release"
 
 sanity_errors=0
 
+printf "Checking that the git repository is in a clean state... "
+if [[ `git status --porcelain` ]] ; then
```

Comment:
Not a blocking comment: I would probably find something like
```suggestion
if [[ $(git status --porcelain | wc -l) -eq 0 ]] ; then
```
a bit less magic but it may be due to my (lack of) bash experience. The `if` and `else` branches would need to be swapped of course for this... (`-eq` instead of `==` only because of other uses in the script, I would actually prefer `==`).

Also note the use of `$()` instead of backticks (just to match other uses in the script). Again, only my subjective opinion, I don't mind either way, just thought I'd point it out.